biome.jsonc

```diff
@@ -1,91 +1,91 @@
 {
-  "$schema": "https://biomejs.dev/schemas/1.7.3/schema.json",
```

Comment:
@balelkin already edited this file if I'm not mistaken

Reply:
I helped him edit this file, but I tested it myself first and left the changes in my branch.